src/effects/backends/builtin/beatgrindeffect.h

```diff
@@ -36,8 +40,12 @@ class BeatGrindState : public EffectState {
     };
 
     mixxx::SampleBuffer loop;
+    mixxx::SampleBuffer reverseLoop;
+    CSAMPLE* selectedLoop;
     int readSamplePos;
     int writeSamplePos;
+    int startPos;
+    int endPos;
```

Comment:
What are the units of these? Samples? Frames? I cannot tell from the variable name or type.

Reply:
firstSample and lastSample are probably the right names for these